de/README.md

```diff
@@ -0,0 +1,28 @@
+# HTTP/3 erklärt
+
+Die Arbeit an diesem Buch wurde im März 2018 gestartet. Es dient der Dokumentation von HTTP/3 und dessen zugrundeliegendem Protokoll: QUIC. Warum, wie sie funktionieren, Protokolldetails, die Implementierung und mehr.
+
+Dieses Buch ist zur Gänze frei zugänglich und wird als kollaborative Angstrengung von jedem getragen, der aushelfen will. 
+
+## Voraussetzungen
+
+Einer Leserin oder Leser wird ein Grundverständnis von TCP/IP, den Grundlagen von HTTP und dem Web vorausgesetzt. Für weitere Einblicke und Details über HTTP/2 empfehlen wir: [http2 explained](https://daniel.haxx.se/http2/).
+
+## Autor
+
+Dieses Buch wurde geschrieben von [Daniel Stenberg](https://daniel.haxx.se/). Daniel ist der Gründer und Lead Developer von [curl](https://curl.haxx.se/), der meistgenutzten HTTP Client Software der Welt. Daniel hat über zwei Jahrzehnte lang mit und an HTTP sowie Internetprotokollen gearbeitet und ist der Autor von [http2 explained](https://daniel.haxx.se/http2/).
+
+## Website
+
+Die Website von diesem Buch kann hier gefunden werden: [daniel.haxx.se/http3-explained](https://daniel.haxx.se/http3-explained).
+
+## Aushelfen
+
+Solltest du Irrtümer, fehlende Details, Fehler oder offensichtliche Lügen in diesem Dokument finden, dann sende uns bitte eine ausgebesserte Version des betroffenen Paragrafen und wir werden diese als neue Version veröffentlichen. Natürlich führen wir alle an, die aushelfen. Ich hoffe, dass wir das Dokument im Laufe der Zeit verbessern können.
+
+Fehler sollten als [Issues](https://github.com/bagder/http3-explained/issues) oder als [Pull Requests](https://github.com/bagder/http3-explained/pulls) auf der Github Seite des Buches übermittelt werden.
```

Comment:
GitHub's brand-name is officially camel-cased (`Github` → `GitHub`). Depends on whether you favor the German grammatically correct version of `Github` or the corporate identity compliant `GitHub`.

Reply:
Thanks! I think for the purpose of accuracy going with the official camel-cased name is better. I pushed the change to the latest pull request.